notebooks/lesson01_intro-to-pandas.ipynb

```diff
@@ -13,7 +13,8 @@
    "cell_type": "markdown",
```

Comment:
<p>isn't is a bit redundant to have "slides" in the file name since the badge already says so? i would prefer<code>lesson01_intro-to-ds-and-ml.pdf</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>